app/javascript/packs/application.js

```diff
@@ -12,6 +12,8 @@ import 'translations/translations';
 import 'initializers/';
 import 'screens/';
 
+let hoang = "test"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/9026022001e8e25853aba46c94d7bfb1f3d5d7bf/app/javascript/packs/application.js#L15"></span>‘hoang’ is assigned a value but never used.</td>
    </tr>
  </tbody>
</table>